frontend/src/components/workspace/Workspace.vue

```diff
@@ -23,12 +23,12 @@
 					ref="documentsPanel"
 				/>
 			</LayoutCol>
-			<LayoutCol class="workspace-grid-resize-gutter" @pointerdown="resizePanel($event)"></LayoutCol>
+			<LayoutCol class="workspace-grid-resize-gutter" @pointerdown="(e) => resizePanel(e)"></LayoutCol>
```

Comment:
What are the benefits of using `(e) => fn(e)` over `fn($event)`?

Reply:
Purely standardization with the rest of the code. It's sort of an arbitrary choice, we could also switch them all to `$event`.